src/client/components/pages/parts/editor-achievements.js

```diff
@@ -125,19 +125,11 @@ class EditorAchievementTab extends React.Component {
 						<DragAndDrop name="rank3"/>
 					</div>
 					<div className="form-group">
-						<span>
-							<button
-								className="btn btn-default"
-								type="submit"
-							>
+						<span style={{marginLeft: '30px'}}>
```

Comment:
With the above change, this margin shouldn't be necessary anymore

Reply:
I tried this, but then the button  would just be a little more left than the above container. So, if you want to make it completely left align, than i can add a margin of just 2-3 px to make sure it is in absolute line. :-)

Reply:
Sure, let's make it fit perfectly !